src/components/editor/Editor.jsx

```diff
@@ -22,23 +22,19 @@ class Editor extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      tempRtState: true,
       resourceTemplateId: '',
       showRdf: false,
       showGroupChooser: false,
     }
   }
 
   componentDidMount() {
-    if (this.state.tempRtState) {
-      if (this.props.location.state !== undefined) {
-        this.setState({
-          resourceTemplateId: this.props.location.state.resourceTemplateId,
-        })
-      } else {
-        this.props.history.push('/templates')
-      }
-      this.setState({ tempRtState: false })
```

Comment:
so presumably it was trying to avoid doing stuff more than once in componentDidMount ...  but spelunking around isn't giving me any more illumination.  I say if there's a problem, it will surface.

Reply:
When this was originally written it was in componentWillUpdate, now that it's in componentDidMount, it doesn't make any sense as that is only called once.

Reply:
ah, ok, that makes sense